src/Symfony/Bundle/FrameworkBundle/Tests/Functional/DebugAutowiringCommandTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Functional;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\ApplicationTester;
+
+/**
+ * @group functional
+ */
+class DebugAutowiringCommandTest extends WebTestCase
```

Comment:
you should use KernelTestCase only, as you don't care about the web part (you won't use a BrowserKit client).

Reply:
I think it's correct actually: this `WebTestCase` lives right in this same `Functional` directory and all the other tests (including those for other commands) use it - it helps setup the temp kernel/project I believe. Changing to `KernelTestCase` made things blow up.

Reply:
ah, I missed that this is the special WebTestCase from the FrameworkBundle testsuite, not the one exposed to users of Symfony